qemu/tests/blockdev_snapshot_install.py

```diff
@@ -24,8 +25,8 @@ def run(test, params, env):
     def tag_for_install(vm, tag):
         if vm.serial_console:
             serial_output = vm.serial_console.get_output()
-            if serial_output and tag in serial_output:
-                return True
+            if serial_output:
+                return re.search(tag, serial_output, re.M) is not None
```

Comment:
```suggestion
            if serial_output and re.search(tag, serial_output, re.M):
                return True
```
whatever the boolean value of `re.search(tag, serial_output, re.M) is not None ` is False or True, the next `logging.info("VM has not started yet")` will not be called.

Reply:
updated